Tasks/VsTestV2/task.json

```diff
@@ -145,6 +145,15 @@
             "helpMarkDown": "Folder to search for the test assemblies.",
             "groupName": "testSelection"
         },
+        {
+            "name": "resultsDirectory",
+            "type": "string",
+            "label": "Test results directory",
+            "defaultValue": "",
```

Comment:
Should we keep the default value as $(Agent.Temp) ?

Reply:
$(Agent.Temp)\TestResults should be the default value

Reply:
Done. The task will take both absolute and relative values. The relative path will be resolved with reference to system.defaultworkingdirectory